Open_Science_New.html

```diff
@@ -164,78 +203,93 @@ <h2><Demo</h2>
 		  <p align="center">Richard Stallman <br>Visit to the MNI<br> May 2017</p>
 		</td>
 		<td width="50%" style="vertical-align:middle">
-		  <img <img src="images/fsf_screenshot.png">
+		  <img src="images/fsf_screenshot.png">
 		       <img src="images/Stallman_Samir.png">
 		</td>
 	    </tr></table>
 	  </section>
-	</section>
 
-	  <!-- Data Publishing -->
-	  <section>
-	    <section data-transition="fade" data-background="images/data_publishing.png">
-	      <h1 style="text-shadow:-1px -1px 0 #000,1px -1px 0 #000,-1px 1px 0 #000,1px 1px 0 #000;"><font color="indigo">Data Publishing<br>
-		- Not Data Sharing -</font></h1>
-	  </section>
-	  <!-- EU going Open Access -->
-	  <section data-transition="fade">
-	    <img src="images/Open_access.png">
-	  </section>
-	  
-	  <!-- FAIR -->
-	  <section data-transition="fade">
-	    <img src="images/FAIR.png">
-	  </section>
-	  
-	  <!-- COBIDAS -->
-	  <section data-transition="fade">
-	    <img src="images/COBIDAS_Nature_new.png">
-	  </section>
-	</section>
-	
-	<!-- Open Science Benefits -->
-	<section data-transition="concave">
-	  
+        <!-- Our implementation -->
 	  <section>
-	    <h2>Benefits of Open Science</h2>
-	    <p>Increased exposure</p>
-	    <p>Greater collaborations</p>
-	    <p>More citations</p>
-	    <p>Less money spent of patents</p>
-	    <p>New funding opportunities</p>
-	    <p>Improved reproducibility</p>
-	    
-	    <p><font color="yellow">Enable scientific discovery</font></p>
-	    <p><font color="yellow">It's the future!</font></p>
-	    <a href="https://www.youtube.com/watch?v=Wtp7R97jz5k" target="_blank"><p><small><font color="orange">Aled Edwards video</font></small></p></a>
+	    <a href="https://samirdas.github.io/MNI_ecosystem.html#/"><img src="images/mni_eco_info_5.jpg" data-background-size="100%"></a>
+          <p><small>View this presentation (click image to open) for further details
+              on our initiatives at the MNI.</small></p>
+	    </section>
+
+          </section>
+
+          <!-- HBHL Column 4-->
+          <section>
+              <h1>HBHL</h1>
+                      <p>Healthy Brains, Healthy Lives</p>
+              <a href="https://www.mcgill.ca/hbhl/"><img src="images/hbhl_logo.png"></a>
+              <p><small><i>Exploring our brains' role in health and disease throughout our lives.</i></small></p>
+              </section>
```

Comment:
@christinerogers Are there any additional slides to include for HBHL? it seems lonely

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/3/2